packages/lab/src/multiline-text/MultilineText.ts

```diff
@@ -0,0 +1,16 @@
+import { Property } from 'csstype';
+import styled from 'styled-components';
+
+export interface MultilineTextProps {
+  overflowWrap?: Extract<
+    Property.OverflowWrap,
+    'anywhere' | 'break-word' | 'normal'
+  >;
+}
+
+export const MultilineText = styled.span<MultilineTextProps>`
```

Comment:
Can we use typography with this component?

Reply:
Yes, you can nest the component inside Typography.